K2Field.K2NE.ServiceBroker/ServiceObjects/Management API/ProcessInstanceManagementSO.cs

```diff
@@ -24,6 +24,9 @@ public override void Execute()
                 case Constants.Methods.ProcessInstanceManagement.GotoActivity:
                     GotoActivity();
                     break;
+                case Constants.Methods.ProcessInstanceManagement.GotoActivity2:
```

Comment:
If we're only ADDING properties, and providing a result. Then i think we can just add that to the existing method. I believe adding properties won't break things if those properties are not required. You can default to the previous behavior maybe?

Reply:
We're changing the method type as well. I'm not sure how changing from Execute to Read will affect the current usages. That's why I added a new method.